packages/kbn-securitysolution-io-ts-list-types/src/common/exception_list/index.ts

```diff
@@ -22,7 +22,7 @@ export type ExceptionListTypeOrUndefined = t.TypeOf<typeof exceptionListTypeOrUn
 export enum ExceptionListTypeEnum {
   DETECTION = 'detection',
   ENDPOINT = 'endpoint',
-  ENDPOINT_TRUSTED_APPS = 'endpoint',
+  ENDPOINT_TRUSTED_APPS = 'endpoint_trusted_apps',
```

Comment:
I don't remember exactly if we filter by the list type somewhere. I know we use most of the times the list_id but just want to make sure we don't break the existing exception list container for trusted apps when upgrading. 
Also wondering if we should include a migration script to update the existing trusted apps container.

Reply:
@FrankHassanabad what else needs to be changed in order for this to work for older TA containers? This change is for the sake of consistency with other artifact names. Actually `endpoint_events` should also be renamed to `endpoint_event_filters`, but we can do that in a new PR if you agree as well.  Thanks for your help on this.

Reply:
@yctercero I've not had a chance to get to it. I'll get to it this week. Thanks for following up.